src/typeMapper.js

```diff
@@ -31,7 +32,11 @@ export function toGraphQL(sequelizeType, sequelizeTypes) {
 
   if (sequelizeType instanceof BOOLEAN) return GraphQLBoolean;
   if (sequelizeType instanceof FLOAT) return GraphQLFloat;
-  if (sequelizeType instanceof INTEGER) return GraphQLInt;
+
+  if (sequelizeType instanceof INTEGER ||
+      sequelizeType instanceof BIGINT) {
```

Comment:
what about the issue with `BIGINT` sometimes being too large to express as a javascript Number?